src/test/kotlin/com/github/jensim/megamanipulator/actions/git/commit/CommitOperatorTest.kt

```diff
@@ -0,0 +1,270 @@
+package com.github.jensim.megamanipulator.actions.git.commit
+
+import com.github.jensim.megamanipulator.actions.ProcessOperator
+import com.github.jensim.megamanipulator.actions.apply.ApplyOutput
+import com.github.jensim.megamanipulator.actions.localrepo.LocalRepoOperator
+import com.github.jensim.megamanipulator.actions.search.SearchResult
+import com.github.jensim.megamanipulator.actions.vcs.PrRouter
+import com.github.jensim.megamanipulator.settings.CodeHostSettings
+import com.github.jensim.megamanipulator.settings.ForkSetting
+import com.github.jensim.megamanipulator.settings.MegaManipulatorSettings
+import com.github.jensim.megamanipulator.settings.SettingsFileOperator
+import com.github.jensim.megamanipulator.ui.DialogGenerator
+import com.github.jensim.megamanipulator.ui.UiProtector
+import com.intellij.openapi.project.Project
+import io.mockk.Called
+import io.mockk.coEvery
+import io.mockk.coVerify
+import io.mockk.every
+import io.mockk.impl.annotations.InjectMockKs
+import io.mockk.impl.annotations.MockK
+import io.mockk.junit5.MockKExtension
+import io.mockk.mockk
+import io.mockk.slot
+import io.mockk.verify
+import kotlinx.coroutines.CompletableDeferred
+import kotlinx.coroutines.runBlocking
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.equalTo
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.NullAndEmptySource
+import org.junit.jupiter.params.provider.ValueSource
+import java.io.File
+import java.nio.file.Path
+import java.util.concurrent.ConcurrentHashMap
+import kotlin.io.path.ExperimentalPathApi
+import kotlin.io.path.createTempDirectory
+import kotlin.test.assertTrue
+
+@ExperimentalPathApi
+@ExtendWith(MockKExtension::class)
+class CommitOperatorTest {
+
+    @MockK
```

Comment:
I don't see the point of using annotations in place of the ` = mockk()`
Are there any benefits to the annotation?
The negatives of using the annotation include
* no compile time feedback
* mutable variables
* 3 lines instead of 1
* no dsl-block-init func

Reply:
I'' change for =mockk() you convinced me